Bindings/Laerdal.Maui.AzureCommunicationCalling.iOS/create_binding.sh

```diff
@@ -11,14 +11,24 @@ cd ..
 
 # Now make a fat framework containing both x86 and arm64 binaries
 cd nativeLibs/Pods/AzureCommunicationCalling/
+rm -rf AzureCommunicationCalling.framework
 mkdir AzureCommunicationCalling.framework
-cp -R AzureCommunicationCalling.xcframework/ios-arm64/AzureCommunicationCalling.framework/* \
+cp -R AzureCommunicationCalling.xcframework/ios-arm64_x86_64-simulator/AzureCommunicationCalling.framework/* \
```

Comment:
Im assuming you tested this on an actual ios device? right? :)
(Just looked like this was a simulator build...)

Reply:
PS and yes this has been tested no real device and simulator ;)